workflow/scripts/osemosys_global/transmission/main.py

```diff
@@ -118,53 +164,57 @@ def main(
     
     # assign capacity to activity unit to transmission + distribution techs
     cap_activity_trn = create_trn_dist_capacity_activity(iar_trn, oar_trn)
-    cap_activity = pd.concat([capact_base, cap_activity_trn]).drop_duplicates(subset=["REGION", "TECHNOLOGY"], keep="last")
+    cap_activity = pd.concat([capact_base, cap_activity_trn]
+                             ).drop_duplicates(subset=["REGION", "TECHNOLOGY"], keep="last")
     
     # OUTPUT CSV's
```

Comment:
Should we be writing out only the `TRN` techs, then having a rule to concat all the different CSVs together. Like the `powerplant/CapitalCosts.csv` only contains power plant capital costs, and the `transmission/CapitalCosts.csv` only contains transmission capital costs. 

That way rules have less dependencies?

Reply:
Also mean you dont need to read in data like capacity to activity units for powerplant technologies